spec/lcm/integration/lcm_end_to_end_spec.rb

```diff
@@ -0,0 +1,255 @@
+      let(:update_scripts) do
+        [
+          {
+            'updateScript' => {
+              'maqlDdl' => "DROP ALL IN {dataset.quotes} CASCADE;\n",
+              'maqlDdlChunks' => ["DROP ALL IN {dataset.quotes} CASCADE;\n"],
+              'preserveData' => true,
+              'cascadeDrops' => true
+            }
+          },
+          {
+            'updateScript' => {
+              'maqlDdl' => "DROP ALL IN {dataset.quotes} CASCADE;\n",
+              'maqlDdlChunks' => ["DROP ALL IN {dataset.quotes} CASCADE;\n"],
+              'preserveData' => false,
+              'cascadeDrops' => true
+            }
+          }
+        ]
+      end
+      let(:client_id_schedule_parameter) { true }
+      let(:user_group) { false }
+      let(:schedule_diff) do
+        [['+', 'hiddenParams.hidden_msg_from_rollout_brick', nil],
+         ['+', 'params.msg_from_rollout_brick', 'Hi, I was set by rollout brick']]
+      end
+    end
+  end
+
+  describe '4 - Modify development project' do
+    it 'modifies development project' do
+      skip 'bug'
+      @project.schedules.each(&:disable!)
+    end
+  end
+
+  describe '5 - Subsequent Release' do
+    before(:all) do
+      @config_template_path = File.expand_path(
+        '../params/release_brick.json.erb',
+        __FILE__
+      )
+    end
+
+    include_context 'release brick'
+
+    it_behaves_like 'a synchronization brick' do
+      let(:original_project) { @project }
+      let(:projects) { $master_projects }
+      let(:schedules_status) { 'DISABLED' }
+      let(:lcm_managed_tag) { false }
+      let(:update_operations) { [] }
+      let(:update_scripts) { [] }
+      let(:client_id_schedule_parameter) { false }
+      let(:user_group) { true }
+      let(:schedule_diff) do
+        [['+', 'hiddenParams.hidden_msg_from_release_brick', nil],
+         ['+', 'params.msg_from_release_brick', 'Hi, I was set by release brick']]
+      end
+    end
+  end
+
+  describe '6 - Subsequent Provisioning' do
+    before(:all) do
+      @config_template_path = File.expand_path(
+        '../params/provisioning_brick.json.erb',
+        __FILE__
+      )
+      LcmHelper.create_workspace_table(
+        @workspace_table_name,
+        @ads_client,
+        Support::CUSTOM_CLIENT_ID_COLUMN
+      )
+      deleted_workspace = @workspaces.delete(@workspaces.sample)
+      @deleted_workspace = @prod_rest_client.projects(:all).find { |p| p.title == deleted_workspace[:title] }
+      @test_context[:input_source_type] = 'ads'
+      # add another workspace to provision
+      @workspaces << {
+          client_id: "INSURANCE_DEMO_NEW_#{@suffix}",
+          segment_id: @workspaces.first[:segment_id],
+          title: "Insurance Demo Workspace NEW #{@suffix}"
+      }
+      # copy existing workspaces ADS as we change data source
+      @workspaces.each do |ws|
+        query = "INSERT INTO \"#{@workspace_table_name}\" VALUES('#{ws[:client_id]}', '#{ws[:segment_id]}', NULL, '#{ws[:title]}');"
```

Comment:
Similar code found in lcm_end_to_end_spec.rb:188, vertica_end_to_end_spec.rb:56 (mass = 40)